Convert HTML comments to PHP, remove empty lines

R33D3M33R opened the pull request:
There are no benefits of exposing code comments to the public. The code was also made more compact.

R33D3M33R committed:
Convert HTML comments to PHP, remove empty lines

There are no benefits of exposing code comments to the public. The code was also made more compact.

mikejolley commented:
Would prefer to just remove them rather than convert to php comments. They don't appear useful or written by ourselves.

R33D3M33R committed:
Update photoswipe.php

Remove comments. Theme developers can read the Photoswipe docs for modification ...

R33D3M33R commented:
I agree. Theme developers can read the Photoswipe docs if they wan't to do modifications. I have pushed a commit the PR to reflect this.
